src/System.Windows.Extensions/tests/System/Drawing/FontConverterTests.cs

```diff
@@ -105,16 +107,27 @@ public void EmptyStringInput()
                 data.Add($"Courier New{s_Separator} Style=Bold", "Courier New", 8.25f, GraphicsUnit.Point, FontStyle.Bold);
                 data.Add($"11px{s_Separator} Style=Bold", "Microsoft Sans Serif", 8.25f, GraphicsUnit.Point, FontStyle.Bold);
 
-                // FullFramework disregards all arguments if the font name is an empty string.
-                if (PlatformDetection.IsWindows10Version1607OrGreater)
+                // empty string is not an installed font on Windows 7, windows 8 and some versions of windows 10.
+                if (EmptyFontPresent)
+                {
+                    data.Add($"{s_Separator} 10{s_Separator} style=bold", "", 10f, GraphicsUnit.Point, FontStyle.Bold);
+                }
+                else
                 {
-                    data.Add($"{s_Separator} 10{s_Separator} style=bold", "", 10f, GraphicsUnit.Point, FontStyle.Bold); // empty string is not a installed font on Windows 7 and windows 8.
+                    data.Add($"{s_Separator} 10{s_Separator} style=bold", "Microsoft Sans Serif", 10f, GraphicsUnit.Point, FontStyle.Bold);
                 }
             }
+            else
+            {
+                // FullFramework disregards all arguments if the font name is an empty string.
+                data.Add($"{s_Separator} 10{s_Separator} style=bold", "Microsoft Sans Serif", 8.25f, GraphicsUnit.Point, FontStyle.Regular);
+            }
```

Comment:
Same question

Reply:
Just to show the contrast between the behavior on .net core and .net framework.

Reply:
We used to do that, but we've been systematically removing all test code paths specific to netfx.  My assumption is at this point we don't want to be adding more.  @ViktorHofer? @danmosemsft?

Reply:
If the src assembly isn't shipping as part of a package on netfx we don't want to add additional .NET Framework validation code in corefx.